backend/requirements.txt

```diff
@@ -21,8 +22,10 @@ django-rest-swagger==2.2.0
 django-webpack-loader==1.0.0
 djangorestframework==3.12.4
 djangorestframework-jwt==1.11.0
+docformatter==1.2
 flake8==3.9.1
```

Comment:
While we're at it might as well update flake8 to 3.9.2 https://pypi.org/project/flake8/